codice/ddf
# Port CDM updates to 2.26.x branch

#### What does this PR do?
This PR includes the following:

- DDF-6408 Improve Logging for CDM
https://github.com/codice/ddf/commit/a68f7b878d97133488c40d6f8a58b02068586969

- Update the contentDirectoryMonitor appender configuration in org.ops4j.pax.logging.cfg to match the configuration in log4j2.xml
https://github.com/codice/ddf/pull/6344

- Cdm Set Max File Size 
https://github.com/codice/ddf/commit/49357927bbf23d7f3a8f8646becd14ab93ef60a3

- CDM timeout update
https://github.com/codice/ddf/commit/1be10fd5e85f0ffc056b203a55671c611b4c7aab

This was tested with Zookeeper 3.5.7, ddf-solr 2.26.0, with DDF simple search.

#### Who is reviewing it? 

@jhunzik 
@figliold 
@clockard 

#### Select relevant component teams: 

#### Ask 2 committers to review/merge the PR and tag them here.
@clockard
@figliold

#### How should this be tested?
- Build this branch, install and configure DDF
- Create a CDM and verify ingest
- Create a file larger than max file size to ingest and verify log
- Create a timeout too small to ingest a large file and verify log

#### Any background context you want to provide?

#### What are the relevant tickets?

#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
